### PR TITLE
RSM domains migration

### PR DESCRIPTION
**Pull request by shaheislamdfe**
### Context

Moving domains from s165 to s189 for frontdoor and DNSZone

### Changes proposed in this pull request

Moving test preprod production environments to production. 

### Guidance to review

Can validate domains are now in s189 via FrontDoor Manager

### Link to Trello card

https://trello.com/c/8gYKfnWH/2131-rsm-migrate-domains-to-s189

### Checklist

- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: terraform/domains/environment_domains/main.tf [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+module "domains" {
+  for_each            = var.hosted_zone
+  source              = "./vendor/modules/domains//domains/environment_domains"
+  zone                = each.key
+  front_door_name     = each.value.front_door_name
+  resource_group_name = each.value.resource_group_name
+  domains             = each.value.domains
+  environment         = each.value.environment_short
+  host_name           = each.value.origin_hostname
+  null_host_header    = try(each.value.null_host_header, false)
```
Comment (saliceti) [inline terraform/domains/environment_domains/main.tf:11]: null_host_header and cached_paths may be removed as we will use the defaults

**Comment by shaheislamdfe** [comment]
[RSM: migrate domains to s189](https://trello.com/c/8gYKfnWH/2131-rsm-migrate-domains-to-s189)

File: Makefile [hunk 2]
```diff
@@ -77,6 +77,18 @@ review: review-init set-azure-resource-group-tags
 	$(eval NAME_ENV=${DEPLOY_ENV}${env})
 	$(eval RESOURCE_ENV=${DEPLOY_ENV}${env})
 
+.PHONY: aks-test
```
Comment (saliceti) [inline Makefile:80]: Why repeat the test preprod and production targets?

Comment (shaheislamdfe) [reply Makefile:80]: Preprod already exists for s165 reference and I didn't want to remove it until the s165 infra is destroyed

**Review by saliceti — APPROVED** [review]

**Pull request merged**